trillian/ctfe/config.go

```diff
@@ -23,9 +24,21 @@ import (
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
 	"github.com/google/certificate-transparency-go/trillian/ctfe/configpb"
+	"github.com/google/certificate-transparency-go/x509"
 	"github.com/google/trillian/crypto/keys/der"
 )
 
+// ValidatedLogConfig represents the LogConfig with the information that has
+// been successfully parsed as a result of validating it.
+type ValidatedLogConfig struct {
```

Comment:
Given this is all inside `trillian/ctfe/`, does the struct and its fields need to be exported?

Reply:
I think it does for when this is imported.

Reply:
Leaving it unexported for now.

Reply:
Aha:
`trillian/ctfe/config.go:101:50:warning: exported func ValidateLogConfig returns unexported type *ctfe.validatedLogConfig, which can be annoying to use (golint)`
Reverting it back to exported.